include/zephyr/bluetooth/gatt.h

```diff
@@ -1507,6 +1507,9 @@ struct bt_gatt_read_params;
 /** @typedef bt_gatt_read_func_t
  *  @brief Read callback function
  *
+ *  When reading using by_uuid, `params->start_handle` is the attribute handle
+ *  for this `data` item.
```

Comment:
Should a similar comment be added for the `start_handle` in `bt_gatt_discover_func_t`? 

IIRC that is also incremented by the stack (in `gatt_discover_next`)

Reply:
In `bt_gatt_discover_func_t`, the handle is already documented to be available in `attr->handle`. And, I think it's better to have only one "correct way" of getting the handle and document that. So, I think we should not document `start_handle` in `bt_gatt_discover_func_t` unless we get rid of `attr->handle`.